### PR TITLE
feat: add moose_developer field

### PR DESCRIPTION
**Pull request by callicles**
* Testing the migration in our dog food
* Adding a property to our instrumentation to distinguish between our devs and external users

### CODE REVIEW

File: apps/framework-cli/src/cli/settings.rs [hunk 1]
```diff
@@ -34,11 +34,21 @@ impl Default for Features {
 #[derive(Deserialize, Debug)]
 pub struct Telemetry {
     pub enabled: bool,
+
+    #[serde(default = "default_moose_developer")]
+    pub moose_developer: bool,
```
Comment (phiSgr) [inline apps/framework-cli/src/cli/settings.rs]: i prefer calling it `is_moose_developer`

File: apps/framework-cli/src/cli/settings.rs [hunk 2]
```diff
@@ -34,11 +34,21 @@ impl Default for Features {
 #[derive(Deserialize, Debug)]
 pub struct Telemetry {
     pub enabled: bool,
+
+    #[serde(default = "default_moose_developer")]
```
Comment (phiSgr) [inline apps/framework-cli/src/cli/settings.rs]: The `Default::default` of `bool` is `false`, I don't think we need this func.

Comment (callicles) [reply apps/framework-cli/src/cli/settings.rs]: I will try

Comment (callicles) [reply apps/framework-cli/src/cli/settings.rs]: You are right!

**Review by phiSgr — APPROVED** [review]

**Pull request merged**